tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -430,6 +430,23 @@ def config_bridge_domain(agent, test_bd, stepinfo='bridge-domain config:')
   end
 end
 
+# Helper for setting system bridge-domain configs
```

Comment:
@chrisvanheuveln 
I tried using dependency_manifest() and this worked ok for manifest and resource tests but the idempotent test failed because the command_config runs every time and there is no output for that. I moved the setup code from utility.rb to test_stp_global.rb file to localize it. Can we wait until the bridge-domain provider to be completed for the dependency_manifest() code to be put in? Can we go ahead with this PR for now?